### ISSUE TITLE
[Keyboard Navigation - Cloud Power Shell - Open Editor] Files tree items like .azure, .cache are not accessible through keyboard.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/monaco-editor#1392]
*User Experience:*
If Files tree items like .azure, .cache are not accessible through keyboard then the users with visual and motion disabilities will not be able to access the tree item controls.

*Repro steps:*
1. Open URL https://ms.portal.azure.com/  and login via valid credentials
2. Navigate to the cloud shell button on the top menu bar and select it.  
3. Cloud Shell popup is launched.
4. Navigate to "Welcome to Azure Cloud Shell" popup and select "PowerShell(Linux)" button.
5. Navigate to "You have no storage mounted" popup window and select "Show advanced settings" button.
6. Navigate to "Subscriptions" dropdown and select "Microsoft Azure" list item and create button.
7. Navigate to "Open Editor" button of "CloudShell-PowerShell Editor" window and select it.
8. Navigate to files and verify the issue.

*Actual Result:*
Files tree items like .azure, .cache are not accessible through keyboard.

*Expected result:*
Files tree items like .azure, .cache should be accessible through keyboard.
Focus indicator should navigate to the tree items and should be able to expand/collapse the tree items.
 
*MAS Reference:*
MAS 2.1.1 - Keyboard (36) - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

![MAS2 1 1_Tree items not accessible](https://user-images.githubusercontent.com/30456910/54994204-b7bb7200-4fe9-11e9-80ce-f34e7e6dcd18.png)

**Comment by sonimishra** [comment]
Issue is copied from Azure Cloud shell - https://msazure.visualstudio.com/One/_workitems/edit/2987812

**Comment by alexdima** [comment]
AFAIK our tree allows for keyboard navigation and expansion of nodes via right arrow. @sonimishra Can you please confirm that right arrow does not expand the tree item?

**Comment by alexdima** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!
